### PR TITLE
Updated Home practice plot, Shiny excel & Session Completion Summaries

### PR DESCRIPTION
**Pull request by cabrinenyona**
Updated Home practice plot, Shiny excel & Session Completion Summaries

### CODE REVIEW

**Comment by cabrinenyona** [comment]
@lilyclements updated the scripts from our working session on home practice and session completion summaries for your review.

File: PfR_loading_data.R [hunk 1]
```diff
@@ -99,5 +99,110 @@ ggplot(session_completion_longer_summary, aes(x = completed, y = `n()`)) +
...
+  group_by(app_user_id) %>%
+  summarise(total_consecutive_sessions = max(count_ones)) %>%
+  mutate(week = current_week) %>%
+  mutate(engagement = ifelse(total_consecutive_sessions < week - 1, "Low",
+                             ifelse(total_consecutive_sessions < week, "Medium",
+                                    ifelse(total_consecutive_sessions == week, "High",
+                                           ifelse(total_consecutive_sessions > week, "Above",
+                                                  "Check")))))
+
+view(session_completion_data_week)
```
Comment (lilyclements) [inline PfR_loading_data.R:158]: to get it to publish, you need to remove or comment out this line `view(...)`

**Review by lilyclements — COMMENTED** [review]
@cabrinenyona looks good! One suggestion in there to see if that fixes your publishing problem :)

**Comment by cabrinenyona** [comment]
@lilyclements .Thanks for pointing that out, I will rectify that. I'm going through the logs to point out and address the errors.
image

**Comment by lilyclements** [comment]
So it says you run View still. (Fourth line from the bottom). Can you remove this line too?

**Pull request merged**